pkg/etc/wavefront/wavefront-proxy/wavefront.conf.default

```diff
@@ -91,71 +68,136 @@ customSourceTags=fqdn, hostname
+## minimum allowed size: pushFlushMaxPoints. Setting this value lower than default reduces memory usage but will force
+## the proxy to spool to disk more frequently if you have points arriving at the proxy in short bursts.
+#pushMemoryBufferLimit=640000
 
-## The following settings are used to configure distributed tracing span ingestion:
+## If there are blocked points, how many lines to print to the log every 10 flushes. Typically 5.
+#pushBlockedSamples=5
+
+## Settings for incoming HTTP request authentication. Authentication is done by a token, proxy is looking for
+## tokens in the querystring ("token=" and "api_key=" parameters) and in request headers ("X-AUTH-TOKEN: ",
+## "Authorization: Bearer", "Authorization: " headers). TCP streams are disabled when authentication is turned on.
+## Allowed authentication methods: NONE, STATIC_TOKEN, HTTP_GET, OAUTH2. Default: NONE
+## - NONE: All requests are considered valid
+## - STATIC_TOKEN: Compare incoming token with the value of authStaticToken setting.
+## - OAUTH2: Validate all tokens against a RFC7662-compliant token introspection endpoint.
+## - HTTP_GET: Validate all tokens against a specific URL. Use {{token}} placeholder in the URL to pass the token
+##             in question to the endpoint. Use of https is strongly recommended for security reasons. The endpoint
+##             must return any 2xx status for valid tokens, any other response code is considered a fail.
+#authMethod=NONE
+## URL for the token introspection endpoint used to validate tokens for incoming HTTP requests.
+## Required when authMethod is OAUTH2 or HTTP_GET ")
+#authTokenIntrospectionServiceUrl=https://auth.acme.corp/api/token/{{token}}/validate
+## Optional credentials for use with the token introspection endpoint if it requires authentication.
+#authTokenIntrospectionAuthorizationHeader=Authorization: Bearer <token>
+## Cache TTL (in seconds) for token validation results (re-authenticate when expired). Default: 600 seconds
+#authResponseRefreshInterval=600
+## Maximum allowed cache TTL (in seconds) for token validation results when token introspection service is
+## unavailable. Default: 86400 seconds (1 day)
+#authResponseMaxTtl=86400
+## Static token that is considered valid for all incoming HTTP requests. Required when authMethod = STATIC_TOKEN.
+#authStaticToken=token1234abcd
+
+############################################# LOGS TO METRICS SETTINGS #################################################
+## Port on which to listen for FileBeat data (Lumberjack protocol). Default: none
+#filebeatPort=5044
+## Port on which to listen for raw logs data (TCP and HTTP). Default: none
+#rawLogsPort=5045
+## Maximum line length for received raw logs (Default: 4096)
+#rawLogsMaxReceivedLength=4096
+## Maximum allowed request size (in bytes) for incoming HTTP requests with raw logs (Default: 16MB)
+#rawLogsHttpBufferSize=16777216
+## Location of the logsingestion.yaml configuration file
```

Comment:
Location of the logsingestion.yaml configuration file
=> Location of the `logsingestion.yaml` configuration file
(putting backticks around file names or code snippets makes things easier to parse.